sonaranalyzer-dotnet/src/SonarAnalyzer.CSharp/Rules/SymbolicExecution/EmptyNullableValueAccess.cs

```diff
@@ -174,5 +141,26 @@ public override IEnumerable<ProgramState> TrySetConstraint(SymbolicValueConstrai
                 return MemberExpression.TrySetConstraint(nullabilityConstraint, programState);
             }
         }
+
+        private sealed class AnalysisContext : ISymbolicExecutionAnalysisContext
```

Comment:
you should move this class declaration above `HasValueAccessSymbolicValue` and `NullValueAccessedCheck`